packages/ng/page-header/page-header.component.html

```diff
@@ -0,0 +1,29 @@
+<header class="pageHeader">
```

Comment:
It seems we have a responsive issue with Page Header actions:

![image](https://github.com/user-attachments/assets/20b7aa3e-d2e3-430b-ab24-93bc5c184200)